### ISSUE TITLE
Set assignee/reviewers, maybe based on commit message?

### ISSUE THREAD

**Issue by amyreese** [ezyang/ghstack#127]
Would be great if I could generate a commit message like this:

```
Summary of changes

Reviewers: @name1 @name2
```

And have ghstack automatically add them as reviewers for the PR.  Even better if there is an option to automatically assign the PR to me, with a separate `Assign-to:` flag in the message to override that behavior.

This would greatly improve the workflow of submitting and reviewing PRs for my team.

**Comment by ezyang** [comment]
Yes I have also found myself wanting this. Setting labels would be good too
